### ISSUE TITLE
Watermark hints do not make sense on the iPad

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#122041]
The watermark hints do not make sense on the ipad since most of those keys can not be pressed on the ipad.
Instead of this we should have button that trigger these commands.

@chrmarti let me know if you do not have time for customising the watermark labels and we can see if somebody else is interested

fyi @JacksonKearl since this might be related to the getting started experience

**Comment by chrmarti** [comment]
@isidorn I'm unlikely to get to this.

**Comment by isidorn** [comment]
@chmarti ok, thanks for letting me know!
Maybe we add a help wanted or see if somebody inside the team would be interested in making changes to the watermarks?

**Comment by JacksonKearl** [comment]
Is the idea that the watermarks would instead be buttons on iPad?

**Comment by isidorn** [comment]
@JacksonKearl yeah I think that is a great idea. That each of those could be touched and just would execute the underlying command.
This change might make sense even for VS Code desktop

**Comment by JacksonKearl** [comment]
I can look into this, there's been talks of getting started interacting with the tips in some ways so it'd be good to look into the code a bit.

**Comment by isidorn** [comment]
@JacksonKearl oh that's wonderful!
If you want we can sync one day for 30 mins to chat about this. Or if you already have your vision on how this should work just let me know once you have something and I can try it out on the iPad!

But having nice buttons which the user can easily touch is a great start
